docs/app/getting-started.md

```diff
@@ -0,0 +1,34 @@
+# Getting Started
```

Comment:
is this stuff still relevant? i saw that it was removed from the top level readme a while ago